scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -48,6 +48,15 @@ sealed trait Execution[+T] extends java.io.Serializable {
   import Execution.{ EvalCache, FlatMapped, GetCounters, ResetCounters, Mapped, OnComplete, RecoverWith, Zipped }
 
   /**
+   * Lift an Execution into a Try
+   *
+   * When this function is called the Execution should never be failed
+   * instead only the Try.
+   */
+  def liftToTry: Execution[Try[T]] =
+    map(e => Success(e)).recoverWith{ case throwable => Execution.from(Failure(throwable)) }
```

Comment:
@richwhitjr I made a second request (use Exception.failed). Was there a reason you didn't take that suggestion?